docs/uma-tokenholders/ulabs-dappmining.md

```diff
@@ -0,0 +1,20 @@
+---
+title: uLABs DaPP Mining Program
```

Comment:
The capitalization on the page looks a bit weird when we have uLABS and dApps. I also think we need to make dApp and uLABS consistent here -> its currently uLABs and DaPP.

Reply:
Thank you! Will stick to uLABS and dApp